README.md

```diff
@@ -18,6 +18,7 @@ The plugin will be loaded by Prettier automatically. No configuration needed. It
 ### Options:
 
 -   `sortingMethod`: `'alphabetical' | 'lineLength' (default)` - What to sort the individual lines by. `alphabetical` sorts by the import path and `lineLength` sorts by the length of the import. Note that alphabetical sorting looks at the **whole** import path, so imports starting with `../` are ranked lower.
+-   `sortingOrder`: `'ascending' | 'descending' (default)` -  Determines the order of imports. If set to 'ascending', imports will be sorted in ascending order, whereas if set to 'descending', they will be sorted in descending order
```

Comment:
Would mention here that it's only available when sorting alphabetically. Also the last line is kind of obvious, would either reword that or remove it